src/main/java/data/InsufficientInfoException.java

```diff
@@ -0,0 +1,21 @@
+package data;
+
+import task.TaskType;
+
+public class InsufficientInfoException extends Exception {
+
+    public InsufficientInfoException(TaskType type) {
+        super(switch (type) {
+            case TODO -> "Oops. task.Task title is required. Please try again with: todo <task.Task Title>.";
+            case EVENT ->
+                    "Oops. Seems like we are missing some information. Please try again with: event <task.Task Title> /from <dd-mm-yyyy hhmm> /to <dd-mm-yyyy hhmm>";
+            case DEADLINE ->
```

Comment:
Do check that your switch statement follows coding convention (switch and case on the same indentation)